pkgs/development/python-modules/limnoria/default.nix

```
@@ -15,10 +24,28 @@ buildPythonPackage rec {
 
   patchPhase = ''
     sed -i 's/version=version/version="${version}"/' setup.py
+
+    # Unix's test suite assumes /bin/ls and /boot exists
+    sed -i 's#/bin/ls#ls#' plugins/Unix/test.py
+    sed -i 's#boot#bin#' plugins/Unix/test.py
```

```suggestion
    sed -i 's|/bin/ls|ls|' plugins/Unix/test.py
    sed -i 's|boot|bin|' plugins/Unix/test.py
```
Did not test this but using the comment character is not a great choice.